### PR TITLE
Update privacy policy

### LINKED ISSUE #1329: Check and deploy new Privacy Policy
I've started drafting amendments to the existing Privacy Policy in accordance with the GDPR requirements. Please see initial copy here: https://docs.google.com/document/d/1LH-j4lUKzJbi-6MQqqEGu3RF_79NQckQobwcohfyZVc/edit# (document is only accessible to mySoc staff).

New text is in blue, and the clauses which each piece of text are attempting to satisfy are in quotation marks in the comments. There are still some areas open to debate (not to mention areas where I've made an educated guess about what we collect/what we do with it), so please weigh in.

This copy covers Research as well as Communications and Democracy so @Rebeccarumbul (and your team) may wish to assess it as well as @JenMysoc and yours.

### LINKED ISSUE #1358: Add link to https://www.mysociety.org/privacy/ next to newsletter subscribe box

### PR DESCRIPTION
**Pull request by zarino**
Fixes #1329 by updating the privacy polict.

Fixes #1358 by adding a link to the _mySociety_ privacy policy near the newsletter signup form in the footer.

### CODE REVIEW

File: www/includes/easyparliament/templates/html/static/privacy.php [hunk 1]
```diff
@@ -13,15 +22,130 @@
             order to run the website.</li>
             </ol>
 
+            <p>If you’d like to know more, read on.</p>
+
+            <h3>What information we collect and how we use it</h3>
+
+            <p><strong>If you sign up to receive alert emails</strong>, we
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/privacy.php]: alert emails link to `/alert/`

File: www/includes/easyparliament/templates/html/static/privacy.php [hunk 2]
```diff
@@ -13,15 +22,130 @@
...
+            information which would identify a specific user. Our Research Data
+            Release policy may be seen on request.</p>
+
+            <p><strong>If you register with the site</strong>, we collect your
+            full name and email address, and — if you opt to provide them — your
+            postcode and the URL of your website.</p>
+            <p>We do not use this data for any purpose other than to allow you
+            to log in to the site.</p>
+
+            <p><strong>If you choose to receive email updates</strong>, we add
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/privacy.php]: This para (and unsubscribe section below) not needed if #1358 done.

File: www/includes/easyparliament/templates/html/static/privacy.php [hunk 3]
```diff
@@ -39,50 +163,13 @@
 
             <p>Google Analytics stores information such as what pages you visit,
             how long you are on the site, how you got here, what you click on,
-            and information about your web browser.</p>
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/privacy.php]: survey questions question not removed a bit above here. And GA text doesn't match.

File: www/includes/easyparliament/templates/html/static/privacy.php [hunk 4]
```diff
@@ -39,50 +163,13 @@
...
-            associate your IP address with any other data held by Google. You
-            may refuse the use of cookies by selecting the appropriate settings
-            on your browser, however please note that if you do this you may not
-            be able to use the full functionality of this website. By using this
-            website, you consent to the processing of data about you by Google
-            in the manner and for the purposes set out above.&rdquo;</p>
-
-            <h3>Opting out</h3>
+            and information about your web browser. IP addresses are masked
+            (only a portion is stored) and personal information is only reported
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/privacy.php]: `       ga('set', 'anonymizeIp', true);` needs adding to header.php

File: www/includes/easyparliament/templates/html/static/privacy.php [hunk 5]
```diff
@@ -39,50 +163,13 @@
...
+            in aggregate. We do not allow Google to use or share our analytics
+            data for any purpose besides providing us with analytics information,
+            and we recommend that any user of Google Analytics does the same.</p>
+
+            <h3>Opting out of Google Analytics tracking</h3>
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/privacy.php]: This section text doesn't match the doc.

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 4 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: Looks like you just added it! https://github.com/mysociety/theyworkforyou/commit/ddfb310bbb9652ffdce331bca48217340a6e4add :+1:

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 5 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: I’ve changed the heading from "Opting out" (in the document) to "Opting out of Google Analytics tracking" to make it clearer what you’re opting out of. The alternative was to make this heading a `<h4>`, but annoyingly I’d also then need to fix the CSS styles to make `<h4>`s appear smaller than `<h3>` (sigh) and just amending the heading text seemed simpler.

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 1 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: Lol. For you, then, I will add a link 😉

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 5 above)
Comment (dracos) [reply www/includes/easyparliament/templates/html/static/privacy.php]: I meant the section (body) text, not the heading text, that's why I said this section text, sorry wasn't clear enough.

Comment (dracos) [reply www/includes/easyparliament/templates/html/static/privacy.php]: But you have changed that so all good.

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 2 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: +1 – happy to remove if @MyfanwyNixon @crowbot agree. At the very least, we’s need to amend the wording of this to make it clear that the "email updates" we’re talking about here are the email _newsletter_ you join via the site footer, and not the "email alerts" you sign up to in various other parts of the website.

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 5 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: Yes, sorry, I knew what you meant. Was just leaving a comment as to why I _hadn’t_ changed the heading text. All good now anyway.

File: www/includes/easyparliament/templates/html/static/privacy.php (hunk 2 above)
Comment (MyfanwyNixon) [reply www/includes/easyparliament/templates/html/static/privacy.php]: Not needed, but it's nice to cover everything IMO. Agree that 'update' would be better as 'newsletter'.

Comment (zarino) [reply www/includes/easyparliament/templates/html/static/privacy.php]: Ok. Updated the wording here, but kept the section.

**Pull request merged**